pkg/eks/eks.go

```diff
@@ -80,11 +80,16 @@ func (c *ClusterProvider) ListClusters(chunkSize int, output string) error {
 	if output == "table" {
 		addListTableColumns(printer.(*printers.TablePrinter))
 	}
-	return c.doListClusters(int64(chunkSize), printer)
+	return c.doListClusters(region, int64(chunkSize), printer)
 }
 
-func (c *ClusterProvider) doListClusters(chunkSize int64, printer printers.OutputPrinter) error {
-	allClusterNames := []*string{}
+type clusterWithRegion struct {
+	Name   *string
+	Region *string
+}
+
+func (c *ClusterProvider) doListClusters(region string, chunkSize int64, printer printers.OutputPrinter) error {
+	allClusterNames := []*clusterWithRegion{}
```

Comment:
could probably use `map[string]string`, but slice of struct is just as fine, as you might want to hang other fields on it eventually...